ru/turing/courses/lesson2/animals/Dog.java

```diff
@@ -0,0 +1,27 @@
+package ru.turing.courses.lesson2.animals;
+
+public class Dog extends Animal {
+    private double hapiness;
+    private boolean is_a_good_boy = true;
```

Comment:
```suggestion
    private boolean isAGoodBoy = true;
```
В джаве пишем с использованием CamelCase. Проверь везде, чтобы через нижние подчеркивания не было.